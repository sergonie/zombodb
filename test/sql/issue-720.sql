select *
from zdb.highlight_document('idxso_posts', '
{
  "my_object_array": [
    {
      "state": "foo"
    },
    {
      "state": "bar"
    },
    {
      "state": "baz"
    }
  ]
}
', 'my_object_array.state = [bar, foo, baz]') order by 1, 2;