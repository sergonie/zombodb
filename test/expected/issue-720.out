select *
from zdb.highlight_document('idxso_posts', '
{
  "my_object_array": [
    {
      "state": "foo"
    },
    {
      "state": "bar"
    },
    {
      "state": "baz"
    }
  ]
}
', 'my_object_array.state = [bar, foo, baz]') order by 1, 2;
      field_name       | array_index | term |    type    | position | start_offset | end_offset |               query_clause                
-----------------------+-------------+------+------------+----------+--------------+------------+-------------------------------------------
 my_object_array.state |           0 | foo  | <ALPHANUM> |        1 |            0 |          3 | my_object_array.state=["bar","foo","baz"]
 my_object_array.state |           1 | bar  | <ALPHANUM> |        1 |            0 |          3 | my_object_array.state=["bar","foo","baz"]
 my_object_array.state |           2 | baz  | <ALPHANUM> |        1 |            0 |          3 | my_object_array.state=["bar","foo","baz"]
(3 rows)

